recipes/gcovr/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "gcovr" %}
+{% set version = "3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c9251b048b1e6cefb065fcfa349f194cb6d8026700b6ab79e8f8376ed245d615
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - gcovr
+
+about:
+  home: http://gcovr.com
+  license: BSD
+  license_family: BSD
+  license_file: 
```

Comment:
Please add the ``license_file``

Reply:
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=license_file#packaging-the-licence-manually